js/components/wysiwyg.js

```diff
@@ -47,9 +47,67 @@ Fliplet.FormBuilder.field('wysiwyg', {
       }
 
       this.value = '';
+    },
+    placeholderLabel: function() {
+      var placeholder_text = this.editor.getElement().getAttribute("placeholder") || this.editor.settings.placeholder;
+      var contentAreaContainer = this.editor.getContentAreaContainer();
+      var defaultStyles = {
+        style: {
+          position: 'absolute',
+          top: '17px',
+          left: '8px',
+          color: '#888',
+          lineHeight: '19px',
+          padding: tinyMCE.DOM.getStyle(contentAreaContainer, 'padding', true),
+          width:'98%',
+          overflow: 'hidden',
+          'white-space': 'pre-wrap',
+          'font-weight': 'normal',
+          'font-size': '16px'
+        }
+      };
+      var placeholder_attrs = this.editor.settings.placeholder_attrs || defaultStyles;
+
+      tinymce.DOM.setStyle(contentAreaContainer, 'position', 'relative');
+
+      // Create label element in the TinyMCE editor
+      this.labelElement = tinymce.DOM.add( contentAreaContainer, this.editor.settings.placeholder_tag || "p", placeholder_attrs, placeholder_text );
+    },
+    placeholderLabelHide: function() {
+      tinymce.DOM.setStyle( this.labelElement, 'display', 'none' );
+    },
+    placeholderLabelShow: function() {
+      tinymce.DOM.setStyle( this.labelElement, 'display', '' );
+    },
+    placeholderOnFocus: function() {
+      if (!this.editor.settings.readonly) {
+        this.placeholderLabelHide();
+      }
+
+      this.editor.execCommand('mceFocus', false);
+    },
+    placeholderOnBlur: function() {
+      if (!this.editor.getContent()) {
+        this.placeholderLabelShow();
+      } else {
+        this.placeholderLabelHide();
+      }
+    },
+    addPlaceHolder: function() {
+      // Init placeholder
+      this.placeholderLabel();
+      this.placeholderOnBlur();
+
+      // Add placeholder listeners
+      tinymce.DOM.bind(this.labelElement, 'click', this.placeholderOnFocus);
+      this.editor.on('focus', this.placeholderOnFocus);
+      this.editor.on('blur', this.placeholderOnBlur);
+      this.editor.on('change', this.placeholderOnBlur);
+      this.editor.on('setContent', this.placeholderOnBlur);
+      this.editor.on('keydown', this.placeholderLabelHide);
     }
   },
-  mounted: function () {
+  mounted: function() {
```

Comment:
Anonymouse functions always have a space between `function` and `()`. Check them all.

```suggestion
  mounted: function () {
```